OMX-27-firmware/sequencer.h

```diff
@@ -157,15 +195,22 @@ int loopCount[NUM_PATTERNS][NUM_STEPS] = {
 	{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
 };
 
-const char* trigConditions[36] = {"1:1","1:2","2:2","1:3","2:3","3:3","1:4","2:4","3:4","4:4","1:5","2:5","3:5","4:5","5:5","1:6","2:6","3:6","4:6","5:6","6:6","1:7","2:7","3:7","4:7","5:7","6:7","7:7","1:8","2:8","3:8","4:8","5:8","6:8","7:8","8:8"};
-int ABcondition = 0;
```

Comment:
this was unused

Reply:
`trigConditions[36]` was there for legendText in `SUBMODE_NOTESEL2` - line 844 in your revisions

Reply:
Oops - sorry, didnt see the replaced version